src/ts/content/extensions/enforceTagIndexAccess.ts

```diff
@@ -9,8 +9,10 @@ import {BackgroundEvent} from "../../common/backgroundEvent";
 
 let canAccessTagIndex = true; // default to true to reduce likelihood of pop in (very unlikely)
 
+const ENFORCEMENT_MODAL_ID = "enforce-tag-index-modal";
+
 const enforceTagIndexAccess = async () => {
-	dismissModals();
+	dismissModals(ENFORCEMENT_MODAL_ID);
```

Comment:
a tad confusing that these are not actually IDs, but i don't think it really matters

Reply:
renaming to `tag`